### ISSUE TITLE
Keep GFX for Matrix visible while scrolling through effects

### ISSUE THREAD

**Issue by FragMenthor** [MoonModules/WLED#81]
**Is your feature request related to a problem? Please describe.**
UI problem related, GFX using matrix is hidden when scrolling through effects

![image](https://github.com/MoonModules/WLED/assets/74264069/b21ba393-8b19-4b28-a19a-4cb49a3bafc5)
![image](https://github.com/MoonModules/WLED/assets/74264069/0cfb2b3a-2aa9-4137-a562-1e37f8cfbbeb)
![image](https://github.com/MoonModules/WLED/assets/74264069/80fd8602-e3a1-47c4-a220-5cea3c43ea90)

**Describe the solution you'd like**
Matrix GFX Should remaing visible when scrolling though effects

**Describe alternatives you've considered**
None.

Thank YOU for making WLED better!

**Comment by dosipod** [comment]
That is a very good idea which we needed as we used this to record demos but unfortunately it was not possible as @ewoudwijma  is a bit busy . The workaround for us is to open another widows with http://wled-ip/liveview2D to record the demos

**Comment by FragMenthor** [comment]
> That is a very good idea which we needed as we used this to record demos but unfortunately it was not possible as @ewoudwijma is a bit busy . The workaround for us is to open another widows with http://wled-ip/liveview2D to record the demos

Yeah, but that, at least for me, is very difficult to use "as is", because of the white background.
For instace, running the "Black Hole" effect in a white background is awfull and cannot be visually "understood":

![image](https://github.com/MoonModules/WLED/assets/74264069/3fff3d3f-c79a-4a66-b586-ee26ba0f3305)

**Comment by dosipod** [comment]
I agree with you ( even worst then you i have issue with bright lights so i must use peek ) , what i ended up doing is install another browser ( i used Firefox developer version ) and setup that with a dark background in Website appearance section 
![image](https://github.com/MoonModules/WLED/assets/87589371/5a19868b-c063-4456-a1a9-e4547b4340d7)

**Comment by softhack007** [comment]
Sometimes there is simply not enough space to keep the preview area visible. 

For example on my phone it looks like this
![Screenshot_20240128_132846](https://github.com/MoonModules/WLED/assets/91616163/d369702f-a3fd-4474-b3fd-4ddb2248ed8c)
